main.nf

```diff
@@ -1236,28 +1244,36 @@ if (params.binning_map_mode == 'all'){
 
 process bowtie2 {
     tag "$assembler-$name"
+    publishDir "${params.outdir}/Assembly/${assembler}/${name}_QC", mode: params.publish_dir_mode,
+        saveAs: {filename -> filename.indexOf(".bowtie2-log") > 0 ? filename : null}
 
     input:
     set val(assembler), val(name), val(group), file(assembly), val(sampleToMap), val(sampleGroup), file(reads) from ch_bowtie2_input
 
     output:
     set val(assembler), val(name), file("${assembler}-${name}-${sampleToMap}.bam"), file("${assembler}-${name}-${sampleToMap}.bam.bai") into ch_assembly_mapping_for_metabat
+    set val(assembler), val(name), val(sampleToMap), file("*.bowtie2-log") into ch_assembly_mapping_stats
```

Comment:
```suggestion
    set val(assembler), val(name), val(sampleToMap), file("*.bowtie2.log") into ch_assembly_mapping_stats
```

Reply:
the file extension would be cleaner I think

Reply:
Agreed, but than it might appear in MultiQC as separate line, I have to test whether it uses everything before a "." as sample name or only before the last ".". I would prefer a less clean file name than a messed up reporting.

Reply:
It actually works, done!